core/src/main/java/org/dcsa/conformance/core/check/JsonSchemaValidator.java

```diff
@@ -44,6 +47,7 @@ private JsonSchemaValidator(String filePath, String schemaName) {
     schemaValidatorsConfig.setTypeLoose(false);
     JsonSchema rootJsonSchema =
         jsonSchemaFactory.getSchema(schemaFileInputStream, schemaValidatorsConfig);
+    schemaFileInputStream.close();
```

Comment:
Use `try-with-resources` instead of manually closing?

Reply:
Otherwise, we fail to `.close()` on error (when `schemaFileInputStream != null && schemaFileInputStream.available() == 0`)

Reply:
Ah yes, of course. I missed that, thanks!